extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -3,3 +3,27 @@
 <div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2">
   <[$ctrl.warningText]>
 </div>
+<!--
+  Forced to use !important here as it makes the code easy to understand, but does not align with design principles.
+  A solution could be added using CSS, but for some reason the styles defined in this file seem to be overriden by Guppy.
+  Only way to fix this is to override the specificity using !important and force the component to take the styles from here
+  and not the styles defined by Guppy.
+-->
+<style>
+  .guppy-div-learner {
+    height: auto !important;
+    width: 100% !important;
```

Comment:
I don't think you need to add `!important` here. These attributes are not being overridden by Guppy. If needed you can make changes to the `guppy_inactive` and `guppy_active` classes.

Also, you might need to add a `min-height`, because rn, the height is shrinking when there is no content:
![ui_bug](https://user-images.githubusercontent.com/35144226/107495050-95b74280-6bb5-11eb-99fe-2d504934b1df.gif)

Reply:
It seems that I still have to override the default height given in the guppy active and guppy inactive classes (it just doesn't work otherwise). I've added the fix for the min-height.